s3fs/tests/test_s3fs.py

```diff
@@ -694,6 +694,19 @@ def test_s3_file_info(s3):
     with pytest.raises(FileNotFoundError):
         s3.info(fn + "another")
 
+def test_content_type_is_set(s3, tmpdir):
+    test_file = os.path.join(str(tmpdir), "test.json")
+    destination = os.path.join(test_bucket_name, "test.json")
+    open(test_file, "w").write("text")
+    s3.put(test_file, destination)
+    assert s3.info(destination)["ContentType"] == "application/json"
+
```

Comment:
I believe the linter will complain if you don't have two lines between functions.